src/Xamarin.Android.Build.Tasks/MSBuild/Xamarin/Android/Xamarin.Android.Device.targets

```diff
@@ -0,0 +1,27 @@
+<!--
+***********************************************************************************************
+Xamarin.Android.Device.targets
```

Comment:
We should add this to the installer but when `'$(_HasCommercialFiles)' != 'True'`

https://github.com/xamarin/xamarin-android/blob/008ba983fa38305284f55be9e8ea0d520d99e992/build-tools/installers/create-installers.targets#L341

So you would get the file in OSS vsix or pkgs, but not the commercial one.

Reply:
@jonathanpeppers: *why* do we want to make this file conditional on OSS vs. commercial?  I think we'll want to migrate e.g. GDB/lldb support here in the future as well, and that would be a *move*, not a "copy", and we'd *certainly* need to have this new file included in *all* installers at that time.

So why not just include it now?

Reply:
As long as `_Run` is replaced and the Debugging targets are imported last, that would be ok.